docs/content/bundles.md

```diff
@@ -125,9 +125,8 @@ metadata. The file should contain a JSON serialized object.
   defaults to `[""]` which means that ALL data and policy must come
   from the bundle.
 
-* OPA will only load data files named `data.json`, i.e., you MUST name files
-  that contain data (which you want loaded into OPA) `data.json` -- otherwise
-  they will be ignored.
+* OPA will only load data files named `data.json` or `data.yaml` (which contain
+  JSON or YAML respectively). Other JSON and YAMl files will be ignored.
```

Comment:
Should we add a note that mentions what kind of yaml is OK? https://github.com/ghodss/yaml/blob/master/yaml.go#L104-L114 specifies which parts are not going to be ok.

Reply:
@patrick-east updated docs to mention limitations on YAML. Also, refactored the manifest section a bit (I think the new version is more useful.) WDYT?

Reply:
Agreed, the updated description is nice.